tests/rptest/services/redpanda.py

```diff
@@ -403,7 +404,9 @@ def __init__(self,
                      int] = None,
                  fast_uploads=False,
                  retention_local_strict=True,
-                 cloud_storage_max_throughput_per_shard: Optional[int] = None):
+                 cloud_storage_max_throughput_per_shard: Optional[int] = None,
+                 cloud_storage_signature_version="s3v4",
+                 before_call_headers=None):
```

Comment:
nit: adding type annotations would be great here

Reply:
fixed